### PR TITLE
v1: use SubElement where possible when creating dom trees (instead of…

### PR DESCRIPTION
**Pull request by deichmab-draeger**
… Element and append)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the [changelog](../CHANGELOG.md) accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: tests/test_mdib.py [hunk 1]
```diff
@@ -64,6 +64,87 @@ def test_get_descriptor_by_code(self):
...
+                 TestData(mdib_text=mdib_dummy.format(delaration.format(my_prefix), '', '', f"{my_prefix}:"),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', '', delaration.format(xsd_prefix), f"{xsd_prefix}:"),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', '', 'xmlns="http://www.w3.org/2001/XMLSchema"', ''),
+                          expected_qname=expected_qname_xsd),
+                 TestData(mdib_text=mdib_dummy.format('', '', delaration_any_uri.format(xsd_prefix), f"{xsd_prefix}:"),
+                          expected_qname=expected_qname_any_uri),
+                 TestData(mdib_text=mdib_dummy.format('', delaration_any_uri.format(xsd_prefix), '', f"{xsd_prefix}:"),
+                          expected_qname=expected_qname_any_uri)]
```
Comment (a-kleinf) [inline tests/test_mdib.py:134]: would suggest to add:
TestData(mdib_text=mdib_dummy.format('xmlns="urn:oid:1.23.3.123.2"', '',  '', ''),
                          expected_qname=expected_qname_any_uri)
                          
 line 144
             prefix = arg_node.text.split(':')[0] if ":" in arg_node.text else None

File: src/sdc11073/mdib/containerproperties.py [hunk 2]
```diff
@@ -450,11 +450,7 @@ def updateXMLValue(self, instance, node):
                 parentNode.remove(subNode)
         else:
             subNode = self._getElementbyChildNamesList(node, self._subElementNames, createMissingNodes=True)
-            if property_value.xml_value is not None:
-                value = property_value.xml_value
-            else:
-                value = namespaces.docNameFromQName(property_value.py_value, subNode.nsmap)
-            subNode.text = value
+            subNode.text = property_value.py_value
```
Comment (a-kleinf) [inline src/sdc11073/mdib/containerproperties.py:453]: if subNode.nsmap.get(None) == property_value.py_value.namespace:
                subNode.text = property_value.py_value.localname
            else:
                subNode.text = property_value.py_value

**Review by a-kleinf — CHANGES_REQUESTED** [review]

**Review by a-kleinf — CHANGES_REQUESTED** [review]

File: tests/test_mdib.py (hunk 1 above)
Comment (deichmab-draeger) [reply tests/test_mdib.py:134]: done

File: src/sdc11073/mdib/descriptorcontainers.py [hunk 3]
```diff
@@ -181,19 +180,20 @@ def _sortedChildNames(self):
...
         :param tag: tag of node, defaults to self.nodeName
         :return: an etree node
         """
         myTag = tag or self.nodeName
-        node = etree_.Element(myTag, attrib={'Handle': self.handle},
+        node = etree_.SubElement(parent_node, myTag, attrib={'Handle': self.handle},
                               nsmap=self.nsmapper.partialMap(Prefix.PM, Prefix.XSI))
-        self._updateNode(node, setXsiType)
         order = self._sortedChildNames()
         self._sortChildNodes(node, order)
```
[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply src/sdc11073/mdib/descriptorcontainers.py:195]: done

**Review by a-kleinf — APPROVED** [review]

**Pull request merged**